WordPress/src/main/java/org/wordpress/android/ui/reader/viewmodels/ReaderViewModel.kt

```diff
@@ -244,9 +242,7 @@ class ReaderViewModel @Inject constructor(
         readerTracker.stop(MAIN_READER)
         wasPaused = true
         if (isChangingConfigurations == false) {
-            updateContentUiState(showQuickStartFocusPoint = false)
```

Comment:
Nitpick : I think we can change the `if (isChangingConfigurations == false)` to `if(!isChangingConfigurations)`. Wdyt? 🤔

Reply:
`isChangingConfigurations` made non-nullable and suggestion applied in 5a30f8c.